src/pages/api/image/ingest.ts

```diff
@@ -5,18 +5,18 @@ import { getServerAuthSession } from '~/server/utils/get-server-auth-session';
 import { ingestImage } from '~/server/services/image.service';
 
 export default async function ingestImageEndpoint(req: NextApiRequest, res: NextApiResponse) {
-  const session = await getServerAuthSession({ req, res });
-  const user = session?.user;
-  if (!user) throw throwAuthorizationError();
-  if (req.method !== 'POST') {
-    res.status(405).send({ message: 'Only POST requests allowed' });
-    return;
-  }
+  if (req.method === 'POST') {
```

Comment:
We have this handy dandy endpoint helper called `AuthedEndpoint` which would take care of checking if the user session is valid and takes a list of allowed methods. We should use that instead